project/ValidatePullRequest.scala

```diff
@@ -134,18 +149,34 @@ object ValidatePullRequest extends AutoPlugin {
 
       val prId = (sourceBranch in ValidatePR).value
 
+      val target = (targetBranch in ValidatePR).value
```

Comment:
Could you add a paragraph to CONTRIBUTING that explains how to use this feature, especially against a different branch?